specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/examples/StorageTargets_CreateOrUpdate.json

```diff
@@ -0,0 +1,107 @@
+            "nfsExport": "exp1",
+            "nfsAccessPolicy": "default"
+          },
+          {
+            "namespacePath": "/path2/on/cache",
+            "targetPath": "/path2/on/exp2",
+            "nfsExport": "exp2",
+            "nfsAccessPolicy": "rootSquash"
+          }
+        ],
+        "targetType": "nfs3",
+        "nfs3": {
+          "target": "10.0.44.44",
+          "usageModel": "READ_HEAVY_INFREQ"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "st1",
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/caches/sc1/storagetargets/st1",
+        "type": "Microsoft.StorageCache/Cache/StorageTarget",
+        "systemData": {
+          "createdBy": "user1",
+          "createdByType": "User",
+          "createdAt": "2020-01-01T17:18:19.1234567Z",
+          "lastModifiedBy": "user2",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2020-01-02T17:18:19.1234567Z"
+        },
+        "properties": {
+          "junctions": [
+            {
+              "namespacePath": "/path/on/cache",
+              "targetPath": "/path/on/exp1",
+              "nfsExport": "exp1",
+              "nfsAccessPolicy": "default"
+            },
+            {
+              "namespacePath": "/path2/on/cache",
+              "targetPath": "/path2/on/exp2",
+              "nfsExport": "exp2",
+              "nfsAccessPolicy": "rootSquash"
+            }
+          ],
+          "targetType": "nfs3",
+          "nfs3": {
+            "target": "10.0.44.44",
+            "usageModel": "READ_HEAVY_INFREQ"
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "name": "st1",
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/caches/sc1/storagetargets/st1",
+        "type": "Microsoft.StorageCache/Cache/StorageTarget",
+        "systemData": {
+          "createdBy": "user1",
+          "createdByType": "User",
+          "createdAt": "2020-01-01T17:18:19.1234567Z",
+          "lastModifiedBy": "user2",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2020-01-02T17:18:19.1234567Z"
+        },
+        "properties": {
+          "junctions": [
+            {
+              "namespacePath": "/path/on/cache",
+              "targetPath": "/path/on/exp1",
+              "nfsExport": "exp1",
+              "nfsAccessPolicy": "default"
+            },
+            {
+              "namespacePath": "/path2/on/cache",
+              "targetPath": "/path2/on/exp2",
+              "nfsExport": "exp2",
+              "nfsAccessPolicy": "rootSquash"
+            }
+          ],
+          "targetType": "nfs3",
+          "nfs3": {
+            "target": "10.0.44.44",
+            "usageModel": "READ_HEAVY_INFREQ"
+          }
+        }
+      }
+    },
+    "202": {}
```

Comment:
This is related to the other comment about 202 Accepted return bodies. The RP does currently return a body, so if the decision is to not return a body on 202 Accepted, then no change needed here. Otherwise, a sample body should be populated.

Reply:
RP Fixed on 2020-10-01 to not return 202s with body.